### ISSUE TITLE
Update firewall zones

### ISSUE THREAD

**Issue by kpengboy** [ocf/puppet#552]
- Output traffic to `scurvy-mgmt` and `dataloss-mgmt` from most hosts should be blocked
- `segfault` should be treated like `tsunami` or `vampires`, i.e. DMZ
- `corruption` should be treated like death, i.e. also DMZ

**Comment by cg505** [comment]
I don't believe `death` is DMZ (except on 80/443) and I don't know why we'd have any reason for `corruption` to be DMZ.

**Comment by kpengboy** [comment]
Those hosts run user code, so we don't trust them as much as internal servers and firewall them off.

(Try ssh'ing from death to other internal hosts e.g. firestorm, you'll see what I mean)

**Comment by cg505** [comment]
My bad, I was understanding it from the other direction (to death from outside the network). This makes sense.

**Comment by dkess** [comment]
`death` is still problematic, and it's something we should fix at around the same time we start sharding user website. The issue is that it needs to bypass the firewall to access ocfweb on Marathon, which shouldn't be exposed to users.
